cmd/spec/example_reservation.go

```diff
@@ -150,6 +150,51 @@ var AzureReservationResponsePayloadDoneExample = payloads.AzureReservationRespon
 	}},
 }
 
+var GCPReservationRequestPayloadExample = payloads.GCPReservationRequestPayload{
+	PubkeyID:           42,
+	SourceID:           "654321",
+	Zone:               "us-east-4",
+	Name:               "my-instance",
+	MachineType:        "e2-micro",
+	Amount:             1,
+	ImageID:            "08a48fed-de87-40ab-a571-f64e30bd0aa8",
+	LaunchTemplateName: "",
+}
+
+var GCPReservationResponsePayloadPendingExample = payloads.GCPReservationResponsePayload{
+	ID:                 1305,
+	PubkeyID:           42,
+	SourceID:           "654321",
+	Zone:               "us-east-4",
+	MachineType:        "e2-micro",
+	Amount:             1,
+	Name:               "my-instance",
+	ImageID:            "08a48fed-de87-40ab-a571-f64e30bd0aa8",
+	LaunchTemplateName: "",
+	GCPOperationName:   "",
```

Comment:
Could you provide example for the operation name and maybe also template name? If we ever commit and push this, we will definitely forget to fill that in when the feature finally lands. Also, operation name I don’t remember what this is for, maybe the example can help.